#this srv-type is set for testing


string function
int64 group
float64 x
float64 y
float64 z
---
string describe
float64 result

